public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -245,6 +263,9 @@ export default class LokiLanguageProvider extends LanguageProvider {
         ...this.labelKeys,
         [EMPTY_SELECTOR]: labelKeys,
       };
+      this.labelValues = {
```

Comment:
This is the actual fix. Not sure if the best place but it seems like calling `start()` is required for things to function anyway.